inji-verify-sdk/README.md

```
@@ -10,98 +31,262 @@ Inji Verify SDK provides ready-to-use **React components** to integrate [OpenID4
-### Step 2: Import and Use
+### Step 2: Choose Your Verification Method
+
+### Option A: QR Code Verification (Scan & Upload)
+
+The QRCodeVerification component enables end-to-end Verifiable Credential (VC) verification using QR codes in Inji-Verify. It supports both camera-based scanning and file upload for QR code verification.
+
+**Perfect for:** Scanning QR codes from documents or uploading QR codes (PNG, JPEG, JPG, PDF).
+
+Steps to integrate:
+
+###  Import & Usage
```

markdownlint-cli2 (0.22.0)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)